frontend/src/components/media_search.svelte

```diff
@@ -98,30 +96,11 @@
       </a>
     {/each}
   </div>
-  <div class="flex space-x-1 justify-center p-1">
-    {#if currentMediaAmount > mediaStartAmount}
-      <button
-        on:click={() => {
-          changeMediaAmount("loadless")
-        }}
-        id="loadless"
-        class="btn"
-      >
-        Show less
-      </button>
-    {/if}
-    {#if currentMediaAmount < meilisearch.nbHits}
-      <button
-        on:click={() => {
-          changeMediaAmount("loadmore")
-        }}
-        id="loadmore"
-        class="btn btn-primary"
-      >
-        Show more
-      </button>
-    {/if}
-  </div>
+  {#if meilisearch.hits.length === meilisearch.limit}
+    <InfiniteLoading on:infinite={loadMoreData} />
+  {:else}
+    <p class="text-center italic">Showing {meilisearch.hits.length} results</p>
+  {/if}
```

Comment:
Not that it should be done here, but we need to modify our API to use offset/pagination to keep this performant. And change the frontend logic to append to the list instead of overriding it

Reply:
#175